portfolio/src/main/webapp/script.js

```diff
@@ -16,8 +16,12 @@
  * Adds a random greeting to the page.
  */
 function addRandomGreeting() {
-  const greetings =
-      ['And I knew exactly what to do. But in a much more real sense, I had no idea what to do.', 'It is a capital mistake to theorize before one has data. Insensibly one begins to twist facts to suit theories, instead of theories to suit facts.', 'When you’re backed against the wall, break the goddamn thing down.', 'Never forget what you are. The rest of the world will not. Wear it like armor, and it can never be used to hurt you.'];
+  const greetings = [
+    'And I knew exactly what to do. But in a much more real sense, I had no idea what to do.',
+    'It is a capital mistake to theorize before one has data. Insensibly one begins to twist facts to suit theories, instead of theories to suit facts.',
+    'When you’re backed against the wall, break the goddamn thing down.',
+    'Never forget what you are. The rest of the world will not. Wear it like armor, and it can never be used to hurt you.'
+  ];
 
```

Comment:
is the greetings still hooked up to the html? If not you can remove this method,